python/ray/tests/test_runtime_env_ray_minimal.py

```diff
@@ -78,7 +80,6 @@ def f():
 
     assert ray.get(f.remote()) == 1
 
-
```

Comment:
Can you revert the changes to this file?

Reply:
Sure